beacon_node/lighthouse_network/src/rpc/mod.rs

```diff
@@ -120,6 +120,8 @@ pub struct RPC<Id: ReqId, TSpec: EthSpec> {
     enable_light_client_server: bool,
     /// Slog logger for RPC behaviour.
     log: slog::Logger,
+    /// ChainSpec for networking constants
+    chain_spec: ChainSpec,
```

Comment:
just store the values you need instead of the whole `ChainSpec`. I believe they are three